configure.ac

```diff
@@ -315,6 +315,14 @@ else
 
 fi
 
+# ---------------------------------------------------------------------
+# MKSQUASHFS
+# ---------------------------------------------------------------------
+AC_CHECK_PROG(MKSQUASHFS_CHECK,mksquashfs,yes)
+if test x"$MKSQUASHFS_CHECK" != "xyes" ; then
+    AC_MSG_ERROR([mksquashfs is required.])
```

Comment:
Would you mind making this warning more like the other warnings in `configure.ac`.  How about this:
```
configure: error: Unable to find mksquashfs, need package squashfs-tools
```
That way a user who doesn't really know how to fix this error doesn't have to Google it.